### DISCUSSION TITLE
Interaction with AllAuth

### DISCUSSION THREAD

**Discussion by danielmcquillen** [nrbnlulu/strawberry-django-auth#576] [Q&A]
Thanks for publishing this great resource. I want to restrict a graphql endpoint to authorized users and was looking for a library that worked similar to Django Rest Framework and it's simple use of tokens.

Question: our site currently uses AllAuth for user login and management. Can we use strawberry-django-auth alongside of that without any interactions?

Thanks for your thoughts!

**Comment by nrbnlulu** [comment]
Hey, I'm not familiar with `AllAuth` but I assume it is possible with some configurations / custom code.
